docs/compute-element/hosted-ce.md

```diff
@@ -1,157 +1,185 @@
+    [contact us](mailto:help@opensciencegrid.org) with the FQDN of yoru login host and the details of your changes.
 
--   Configure central OSG services to schedule jobs
 
-Optionally, we can assist you in
-[installing and setting up the Squid and OASIS](#optional-providing-access-to-application-software-using-oasis)
-software on your cluster in order to support application software repositories.
-This will allow a broader set of jobs to run on your cluster.
+Finalizing Installation
+-----------------------
 
+After applying for an OSG Hosted CE, our staff will contact you with the following information:
 
-Providing the OSG Worker Node client (HTCondor batch systems only)
-------------------------------------------------------------------
+-   IP ranges of OSG hosted services
+-   Public SSH key to be installed in the [OSG accounts](#user-accounts)
 
-All OSG sites need to provide the OSG Worker Node Client on each worker node in the cluster.
-This is normally handled by OSG staff for a Hosted CE, but requires shared home directories across the cluster.
+Once this is done, OSG staff will work with you and your team to begin submitting resource requests to your site, first
+with some tests, then with a steady ramp-up to full production.
 
-However, for sites with an HTCondor batch system, often there is no shared filesystem set up.
-If you run an HTCondor site and it is easier to install and maintain the Worker Node Client on each worker node than to
-set up and maintain shared file system, you have the following options:
 
--   Install the [Worker Node Client from RPM](../worker-node/install-wn.md)
--   Install the [Worker Node Client from tarball](../worker-node/install-wn-tarball.md)
--   Install the Worker Node Client from [OASIS](../worker-node/install-wn-oasis.md)
+### Validating contributions ###
 
+In addition to any internal validation processes that you may have, the OSG provides monitoring to view which
+communities and projects within said communities are accessing your site, their fields of science, and home institution.
+Below is an example of the monitoring views that will be available for your cluster.
 
-**Optional**: Providing Access to Application Software Using OASIS
-------------------------------------------------------------------
+![monitoring graphs](img/monitoring_graphs.png)
 
-Many OSG jobs make of use software modules provided by their collaborations or by the OSG User Support team.
-In order to support these jobs without having to install specific software modules on your cluster,
-OSG sites may provide a distributed software repository system called OASIS, built on top of
-[CVMFS](https://cernvm.cern.ch/portal/filesystem).
+To view your contributions, select your site from the `Facility` dropdown of the
+[Payload job summary](https://gracc.opensciencegrid.org/d/000000043/pilot-jobs-summary?orgId=1) dashboard.
+Note that accounting data may take up to 24 hours to display.
```

Comment:
Maybe "up to 24 hours to update"? "24 hours to display" makes it sound like the website is _really slow_.

Reply:
Sure, sounds good. Though I think @djw8605 mentioned on software-discuss that usage should show up after 1 hr

Reply:
New registrations take up to 24 hours to show up in GRACC.  Topology is only read every 24 hours by the summarizing process.  But, the usage should show up in ~30 minutes, if the site was already registered 24+ hours ago.